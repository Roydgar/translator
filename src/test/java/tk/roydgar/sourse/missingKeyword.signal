myprog;
const ident = 10;

begin
end.
