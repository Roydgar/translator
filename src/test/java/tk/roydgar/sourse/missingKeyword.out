   1    1        401    myprog
   1    7          3    ;
   2    1        304    const
   2    7        402    ident
   2   13          0    =
   2   15        501    10
   2   17          3    ;
   4    1        302    begin
   5    1        303    end
   5    4          5    .

Tree:
<signal-program>
    <program>
        PROGRAM
        <procedure-identifier>
            <identifier>
                
        ;
        <block>
            <declarations>
                <constant-declaration>
                    <empty>
            BEGIN
            <statement-list>
                <empty>
            END
        .
